features/OCM_insights_tab.feature

```diff
@@ -0,0 +1,90 @@
+Feature: Insights tab on OCM UI
+
+  Scenario: Insights tab for cluster without any issue
+    Given console.redhat.com is accessible
+      And user U1 is already loged in into console.redhat.com
+      And user U1 switch to "OpenShift" page in console.redhat.com (OCM UI)
+      And user U1 is part of account (organization) at least one cluster C1
+      And the cluster C1 sent data to the external data pipeline
+      And no issues have been detected for cluster C1
+     When user U1 selects "Clusters" item from the menu
+     Then new page with list of clusters should be displayed
+      And the page should contain at least one cluster
+      And button "Create cluster" should be made visible
+      And button "Register cluster" should be made visible
+     When user U1 click on cluster name
+     Then a page with detailed information about the selected cluster should be displayed
+      And at least four tab headers should be displayed on that page
+          | Tab name         |
+          | Overview         |
+          | Monitoring       |
+          | Insights Advisor |
+          | Support          |
+     When user U1 clicks on tab named "Insights Advisor"
+     Then new page should be displayed saying "Your cluster passed all recommendations"
```

Comment:
This is what I see

<img width="1144" alt="image" src="https://user-images.githubusercontent.com/15891027/137877790-ebdcb904-55a5-40d8-9ee5-9159995e9d71.png">